app/src/main/java/org/mifos/mobile/ui/activities/PassCodeActivity.kt

```diff
@@ -100,27 +134,7 @@ class PassCodeActivity : MifosPassCodeActivity() {
     }
 
     override fun startLoginActivity() {
-        MaterialDialog.Builder().init(this@PassCodeActivity)
-            .setCancelable(false)
-            .setMessage(R.string.login_using_password_confirmation)
-            .setPositiveButton(
-                getString(R.string.logout),
-                DialogInterface.OnClickListener { _, _ ->
-                    val i = Intent(
-                        this@PassCodeActivity,
-                        LoginActivity::class.java,
-                    )
-                    i.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
-                    startActivity(i)
-                    finish()
-                },
-            )
-            .setNegativeButton(
-                getString(R.string.cancel),
-                DialogInterface.OnClickListener { dialog, _ -> dialog.dismiss() },
-            )
-            .createMaterialDialog()
-            .show()
+        TODO()
```

Comment:
why TODO() ?

Reply:
Because earlier the show dialogue box logic was written in this method, so I moved that logic to button click listener and may be in future we could write some appropriate functionality in this overwritten method therefore I mentioned TODO in it